quartz/i18n/locales/nl-NL.ts

```diff
@@ -82,5 +82,16 @@ export default {
         count === 1 ? "Eerste label tonen." : `Eerste ${count} labels tonen.`,
       totalTags: ({ count }) => `${count} labels gevonden.`,
     },
+    encryptedContent: {
+      loading: "Bezig met laden...",
+      password: "Wachtwoord",
+      submit: "Verzenden",
+      enterPassword:
+        "Deze pagina is standaard vergrendeld. Voer het wachtwoord in om te ontgrendelen:",
+      modernBrowser: "Gebruik een moderne browser.",
+      wrongPassword: "Nieprawidłowe hasło. Wprowadź ponownie hasło, aby odblokować:",
+      noPayload: "Er is geen versleutelde payload.",
+      decrypting: "Decrypteren...",
```

Comment:
The proper Dutch term here is `Ontsleutelen...`

Reply:
I will use your translated version for it